src/Codeception/Module/REST.php

```diff
@@ -608,7 +615,20 @@ protected function execute($method, $url, $parameters = [], $files = [])
         if ($this->isBinaryData($printedResponse)) {
             $printedResponse = $this->binaryToDebugString($printedResponse);
         }
-        $this->debugSection("Response", $printedResponse);
+
+        $short = $this->_getConfig('shortDebugResponse');
+
+        if ($short !== null && $short !== false) {
+            if (!is_int($short) && $short <= 0) {
+                $short = $this->DEFAULT_SHORTEN_VALUE;
```

Comment:
It would be better to perform validation and normalization of `shortDebugResponse` parameter in `validateConfig` method.
You have to add such method to REST class and call `parent::validateConfig()` before doing your own logic.
https://github.com/Codeception/Codeception/blob/2.5.5/src/Codeception/Module.php#L148

@DavertMik is validateConfig the right place for this code?

Reply:
I think yes

Reply:
Sure, I try to solve all comments above next week and push a new MR.